PANDORA/PMHC/Model.py

```diff
@@ -10,8 +10,8 @@
 
 class Model:
```

Comment:
why does "Model" need an output_dir argument now?

Reply:
Which part are you referring to? Or do you mean in the whole Model class?

Reply:
Oh nevermind. I thought you added it with this PR and I could not see why.

Reply:
I have removed output_dir from the model class and the code still works. Should I put output_dir back or not?